website/next.config.js

```diff
@@ -5,7 +5,7 @@ console.log(`Environment: ${process.env.HASHI_ENV}`)
 
 module.exports = withHashicorp({
   defaultLayout: true,
-  transpileModules: ['is-absolute-url'],
+  transpileModules: ['is-absolute-url', '@hashicorp/react-image'],
```

Comment:
Confused about this one, don't we transpile all the global components by default now?

Reply:
We should, but this dependency was nested so it didn't get picked up by my solution.  I'll add a ticket to handle this case within scripts.

Reply:
Ticket: https://app.asana.com/0/1100423001970639/1199916284212909/f